metric_tracking/__init__.py

```diff
@@ -15,12 +16,13 @@ def do_tracking():
         mlflow.log_param("Date A", date_a)
         mlflow.log_param("Date B", date_b)
         calculate_score(date_a, date_b)
+        calculate_metrics(date_a,date_b)
         mlflow.end_run()
-
+    
```

Comment:
@tumido do we have a bot commenting on style? If so, can we get it enabled in this repo?

Reply:
@durandom We currently use a Travis for that. One step in its job is to do the linting. That's what you mean? Sure, we can enable it. :) It you'd like to integrate a bot pointing out the violations directly in a comment, we currently don't have that enabled/available, but I think AI-CoE bots do something like that, don't they?

Reply:
can't miq-bot do python? @juliancheal ?
@goern - do your linters post comments in the PR?